roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -14,6 +14,25 @@ assetConfig:
 {% if 'metrics_public_url' in openshift.master %}
   metricsPublicURL: {{ openshift.master.metrics_public_url }}
 {% endif %}
+{% if 'extension_scripts' in openshift.master %}
+  extensionScripts:
```

Comment:
We have a PR to add a filter for padding yaml: https://github.com/openshift/openshift-ansible/pull/1148
Once that is merged, this could become:
```
{% if 'extension_scripts' in openshift.master %}
{{ openshift.master.extension_scripts | to_padded_yaml(1, 2) ""
{% endfor %}
```